scripts/check_configs.ts

```diff
@@ -594,6 +622,12 @@ async function main() {
         }
       }
 
+      if(jsonData.market_banners) {
+        if(!isValidMarketBanners(jsonData.market_banners, network, marketIds)) {
+          outcomeMap[network] = false;
+        }
+      }
```

Comment:
If possible, please combine the nested if statements into 1 if statement to avoid nesting:
```typescript
if (jsonData.market_banners && !isValidMarketBanners(jsonData.market_banners, network, marketIds)) {
  outcomeMap[network] = false;
}
```